schema/bigquery/SessionAnswer.js

```diff
@@ -103,10 +103,10 @@ cube(`AggregateSessionMetrics`, {
       ROW_NUMBER() OVER (ORDER BY plio_id, user_id) as id,
       plio_id,
       user_id,
-      COUNT(case when question_type = 'mcq' AND answer IS NULL THEN NULL when question_type = 'subjective' AND answer IS NULL then NULL else 1 end) AS num_answered,
+      COUNT(case when question_type IN ('mcq', 'checkbox') AND answer IS NULL THEN NULL when question_type = 'subjective' AND answer IS NULL then NULL else 1 end) AS num_answered,
```

Comment:
Why not this?
```suggestion
      COUNT(case when question_type IN ('mcq', 'checkbox', 'subjective') AND answer IS NULL THEN NULL else 1 end) AS num_answered,
```

Reply:
Also let's keep it consistent
SQL language strings should stay in uppercase
```suggestion
      COUNT(CASE WHEN question_type IN ('mcq', 'checkbox') AND answer IS NULL THEN NULL WHEN question_type = 'subjective' AND answer IS NULL THEN NULL ELSE 1 END) AS num_answered,
```